### PR TITLE
feat: [CDS-42905]: confirmDelete, useInterval hook with compute current time between the range 

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

- Confirm Delete
- Create useInterval hook. Check active state of the freeze window between the range

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 15 Issues 
* Code issue 15 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11602)

**Review by ishantmehtaharness — APPROVED** [review]

File: src/modules/72-freeze-windows/hooks/useConfirmFreezeDelete.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        action(deleteId)
+      }
+      setDeleteId(undefined)
+    }
+  })
+
+  const confirmFreezeDelete = (data?: string) => {
+    setDeleteId(data)
+    openDialog()
+  }
```
Comment (udayvunnam) [inline src/modules/72-freeze-windows/hooks/useConfirmFreezeDelete.ts:34]: needed to pass the data to and get it from the dialog

File: src/modules/72-freeze-windows/hooks/useCurrentActiveTime.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import moment from 'moment'
+import { useMemo, useState } from 'react'
+import { useInterval } from '@common/hooks/useInterval'
+
+const ONE_MINUTE = 60_000
```
Comment (udayvunnam) [inline src/modules/72-freeze-windows/hooks/useCurrentActiveTime.ts:12]: currently kept as a minute since we don't support the seconds is freeze time

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**